easybuild/framework/easyblock.py

```diff
@@ -1445,7 +1445,20 @@ def extensions_step(self, fetch=False):
 
     def package_step(self):
```

Comment:
this method should be moved elsewhere (a `package.py` module perhaps?)  and be called `package_fpm`.
then call this new method from here. the packaging flavour should be selected via configuration option (so might wrap this in a rather abstract `Packaging` class in said `package` module; but a simple function will do for now)